docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.6.2
+
+MAINTAINER Michał Czeraszkiewicz <contact@czerasz.com>
+
+# Set the reset cache variable
+ENV REFRESHED_AT 2016-05-10
```

Comment:
Find more details [here](http://czerasz.com/2014/11/13/docker-tip-and-tricks/#use-refreshedat-variable-for-better-cache-control).